how-to/how-to-import-kata-logs-with-fluentd.md

```diff
@@ -0,0 +1,604 @@
+# Importing Kata Containers logs with [`fluentd`](https://www.fluentd.org/)
+
+* [Introduction](#introduction)
+* [Overview](#overview)
+    * [Test stack](#test-stack)
+    * [Importing the logs](#importing-the-logs)
+    * [Direct import `logfmt` from `systemd`](#direct-import-logfmt-from-systemd)
+        * [Configuring `minikube`](#configuring-minikube)
+        * [Pull from `systemd`](#pull-from-systemd)
+        * [Summary](#summary)
+    * [Next steps](#next-steps)
+        * [JSON in files](#json-in-files)
+            * [Prefixing all keys](#prefixing-all-keys)
+* [Caveats](#caveats)
+* [Summary](#summary)
+
+# Introduction
+
+This document describes how to import Kata Containers logs into `fluentd`, typically
+for importing into an
+[EFK](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/fluentd-elasticsearch#running-efk-stack-in-production)
+or [ELK](https://www.elastic.co/elastic-stack) stack.
+
+> **Note:** This document does not cover any aspect of 'log rotation'. It is expected that any production
```

Comment:
Can you change single quotes to double-quotes (here and below). See https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#the-apostrophe.